mockServiceWorker.js

```diff
@@ -79,10 +93,14 @@ self.addEventListener('fetch', async (event) => {
         cache: req.cache,
         mode: req.mode,
         credentials: req.credentials,
+        destination: req.destination,
+        integrity: req.integrity,
         redirect: req.redirect,
         referrer: req.referrer,
         referrerPolicy: req.referrerPolicy,
         body: json || text,
+        bodyUsed: req.bodyUsed,
+        keepalive: req.keepalive,
```

Comment:
Get keepAlive is more appropriate, just feel the style is unified some

Reply:
In terms of naming consistency I agree, but this is a property that is spelled that way (see the "Browser compatibility" table in [Request](https://developer.mozilla.org/en-US/docs/Web/API/Request) documentation).